package.json

```diff
@@ -203,16 +203,16 @@
     "qrcode": "1.4.4",
     "qs": "6.9.7",
     "querystring-es3": "0.2.1",
-    "react": "18.2.0",
+    "react": "19.0.0",
     "react-coin-icon": "rainbow-me/react-coin-icon#06464588a3d986f6ef3a7d7341b2d7ea0c5ac50b",
     "react-fast-compare": "3.2.2",
     "react-flatten-children": "1.1.2",
     "react-freeze": "1.0.4",
-    "react-native": "0.74.3",
+    "react-native": "0.79.0",
     "react-native-action-sheet": "2.2.0",
     "react-native-actionsheet": "2.4.2",
     "react-native-aes-crypto": "rainbow-me/react-native-aes#65c49f7e70266615b2999eaa7db654d3fe4f2e3b",
-    "react-native-animateable-text": "0.12.1",
+    "react-native-animateable-text": "0.15.0",
```

Comment:
99% sure we need `^0.16.0-beta.0` for `0.79` compatibility.

Reply:
https://github.com/axelra-ag/react-native-animateable-text?tab=readme-ov-file#compatibility